scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
+Copyright (c) AppDynamics, Inc., and its affiliates 2015
+All rights reserved
+
+Build script for wpt-driver.
+
+REQUIREMENTS:
+- msbuild.exe and python.exe (required by the Chrome compile.cmd) should be in
+  the path
+
+This script has been tested in GitHub's Git Shell.
+Executing it in a regular PowerShell may also work, but you will have to
+setup the path for each dependencies.
+#>
+
+param(
+    [string]$configuration = "Release"
+)
+
+$wptroot = $PSScriptRoot + "\.."
+
+"*** Building wpt-driver with $configuration configuration"
+
+# ZipFiles implementation from:
+# http://stackoverflow.com/a/13302548
+function ZipFiles( $zipFilename, $sourceDir)
+{
+   Add-Type -Assembly System.IO.Compression.FileSystem
+   $compressionLevel = [System.IO.Compression.CompressionLevel]::Optimal
+   [System.IO.Compression.ZipFile]::CreateFromDirectory($sourcedir,
+        $zipfilename, $compressionLevel, $false)
+}
+
+$releasedir = Get-ChildItem $wptroot $configuration
+$outdir = ($releasedir.fullname + "\dist")
+
+# Create output directories
+if (test-path $outdir) { remove-item -recurse $outdir }
+mkdir $outdir | Out-Null
+mkdir ($outdir + "\extension") | Out-Null
+mkdir ($outdir + "\extension\templates") | Out-Null
+
+"-> Building the Visual Studio Solution in $configuration configuration"
+"(log file is $wptroot\$configuration\msbuild.log)"
+msbuild.exe webpagetest.sln /P:Configuration=$configuration | Out-File ($outdir + "\msbuild.log")
+
+# Copying compiled stuff into the output directory
+$binaries = "wptbho.dll", "wptdriver.exe", "wpthook.dll", "wptload.dll", "wptupdate.exe", "wptwatchdog.exe"
+foreach ($bin in $binaries) {
+	Get-ChildItem -Path $wptroot\$configuration $bin | Copy-Item -Destination $outdir
+}
+
+"-> Building and copying Chrome extension"
+"(log file is $wptroot\$configuration\msbuild.log)"
+& $wptroot\agent\browser\chrome\compile.cmd 2>&1 | Out-File ($outdir + "\chromebuild.log")
```

Comment:
there seems to be some problem with this line. When I run it in powershell, everything works, but I see this error:
```
PS C:\Users\Public\webpagetest\Debug\dist> cat .\chromebuild.log
compile.cmd : C:\Users\Public\webpagetest\agent\browser\chrome\extension/third_party/closure-library/closure/bin/build/
closurebuilder.py: Scanning paths...
At C:\Users\Public\webpagetest\scripts\create-update.ps1:55 char:1
+ & $wptroot\agent\browser\chrome\compile.cmd 2>&1 | Out-File ($outdir + "\chromeb ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (C:\Users\Public...anning paths...:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError

C:\Users\Public\webpagetest\agent\browser\chrome\extension/third_party/closure-library/closure/bin/build/closurebuilder
.py: 698 sources scanned.

C:\Users\Public\webpagetest\agent\browser\chrome\extension/third_party/closure-library/closure/bin/build/closurebuilder
.py: Building dependency tree..
```